<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevicesWnd</class>
 <widget class="QWidget" name="DevicesWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Devices</string>
  </property>
  <widget class="QTableWidget" name="tableDevices">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>700</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="incrCol">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加列</string>
   </property>
  </widget>
  <widget class="QPushButton" name="incrRow">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decrRow">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>减少行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decrCol">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>减少列</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connAll">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>210</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>  连接所有Qbox</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnAll">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>250</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>  断开所有Qbox</string>
   </property>
  </widget>
  <zorder>tableDevices</zorder>
  <zorder>incrRow</zorder>
  <zorder>decrRow</zorder>
  <zorder>decrCol</zorder>
  <zorder>incrCol</zorder>
  <zorder>connAll</zorder>
  <zorder>disconnAll</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
